examples/uv_actor.lua

```diff
@@ -0,0 +1,48 @@
+-- Use LibUV event loop in actor thread
+--
+-- !NOTE! I can not figure out why this does not work
+--  with `inproc` transport so I use `tcp` to communicate
+--  with actor thread
```

Comment:
yes i believe inproc does not signal changes is socket state. 

but you're saying it works if you do a pipe:recv() in the parent and a pipe:send() in the child? Does it work if change it so the parent "connect" and the child "bind"